packages/arb-util/configuration/configuration.go

```diff
@@ -160,6 +161,13 @@ type Node struct {
 	WS         WS         `koanf:"ws"`
 }
 
+type NodeCache struct {
+	AllowSlowLookup  bool          `koanf:"allow-slow-lookup"`
```

Comment:
This needs to be surfaced as a command line option

Reply:
:+1: